src/ApolloTest.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+
+const EXCHANGE_RATES = gql`
+  {
+    rates(currency: "USD") {
+      currency
+      rate
+    }
+  }
+`;
+
+// prints dollar exchange rates from Conbase API
+export function ApolloTest() {
```

Comment:
how are you running this test? it's not coming up with the command "yarn test" (though an outdated CRA test I never got rid of is, oops)

Reply:
It wasn't a proper test. It was just a component which if used pulled data from Coinbase Api. Just an example how to get data with GraphQL for my future reference :D Thinking now, it shouldnt be commited at all, so I've deleted it.